packages/ui/src/components/ChatHistorySidebar/chat-history-rail.css

```
@@ -0,0 +1,127 @@
+		oklch(85% 0 0) var(--rail-top-fade) calc(100% - var(--rail-bottom-fade)),
+		oklch(60% 0 0 / 0)
+	);
+	animation-name: chat-history-rail-edge-fade;
+	animation-timing-function: linear;
+	animation-fill-mode: both;
+	animation-timeline: scroll(self y);
+}
+
+.chat-history-rail-marker {
+	--marker-progress: 0;
+	width: 26px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)